assets/tests/factories/shape.ts

```diff
@@ -0,0 +1,9 @@
+import { Factory } from "fishery"
+import { Shape } from "../../src/schedule"
+import stopFactory from "./stop"
+
+export default Factory.define<Shape>(({ sequence }) => ({
+  id: `shape${sequence}`,
+  points: [{ shape_id: `shape${sequence}`, sequence: 1, lat: 0, lon: 0 }],
```

Comment:
question: why is `sequence` and `shape_id` being provided to `ShapePoint`?

Reply:
Looks like that was just a mistake on my part - fixed in c2b29a82.